### PR TITLE
Add Space Systems Engineering 14th Edition as reference.

### LINKED ISSUE #85: add reference for "Space Systems Engineering"
Textbook reference Space Systems Engineering 14th Ed. was used to specify data structure for a "satellite health" packet #80

### PR DESCRIPTION
**Pull request by kchiu002**
Close #85

### CODE REVIEW

File: architecture/0-References/6-Space_Systems_Engineering.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+title: Space Systems Engineering 14th Ed. 
+url: ''
```
Comment (capsulecorplab) [inline architecture/0-References/6-Space_Systems_Engineering.yml:2]: url's empty

Comment (kchiu002) [reply architecture/0-References/6-Space_Systems_Engineering.yml:2]: Since it's a textbook, I am unsure as to what URL to place here.

Comment (capsulecorplab) [reply architecture/0-References/6-Space_Systems_Engineering.yml:2]: Fair point. I don't suppose there's an online ebook somewhere?

**Comment by capsulecorplab** [comment]
PR should be updated to meet content style guide

**Review by capsulecorplab — APPROVED** [review]
LGTM

**Pull request merged**